fastlane/Constants/Constant.swift

```diff
@@ -41,6 +43,7 @@ enum Constant {
     // MARK: - Symbol
 
     static let uploadSymbolsBinaryPath: String = "./Pods/FirebaseCrashlytics/upload-symbols"
+    static let dsymSuffix: String = ".dSYM.zip"
```

Comment:
what do you think about using `dSYMSuffix` as in `dSYM` vs `dsym`

Reply:
Good catch! Let's me update :D

Reply:
Updated in https://github.com/nimblehq/ios-templates/pull/361/commits/766ea85be36e38584f968badd381a86f8ec6d738